### ISSUE TITLE
Have u seen colorbox.io?

### ISSUE THREAD

**Issue by otobrglez** [some1else/perceptualab#1]
Check it out for some inspiration.

[colorbox.io](https://www.colorbox.io/)

✌️ 

- Oto

**Comment by some1else** [comment]
Yeah I tried to replicate the UI for the purpose of building a palette of
perceptually smooth, equidistant colors (like an interactive version of the
http://some1else.github.io/life gradient). But I see now that this approach is
flawed. Aside from needless easing curves, that can never fix the fact that
the interpolation is done in RGB... That's just not how you build a palette. This
things needs stop-colors and a wheel
